### PR TITLE
feat:added wishlist page

### PR DESCRIPTION
**Pull request by abhishek17551**
Screenshot (133)
Screenshot (134)

### CODE REVIEW

File: pages/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+.full-wrapper h3{
```
Comment (tushargoyal812) [inline pages/wishlist/wishlist.css]: suggestion: add space before {

**Review by tushargoyal812 — COMMENTED** [review]
minor changes needed,looks good to merge.

File: pages/wishlist/wishlist.css (hunk 1 above)
Comment (abhishek17551) [reply pages/wishlist/wishlist.css]: sure thing. will change henceforth.

**Review by sharmayatendra — APPROVED** [review]
looks good to merge.

**Pull request merged**